docs/content/en/usage.md

```diff
@@ -48,7 +48,7 @@ Axios plugin also supports shortcuts with `$` prefixed methods to directly get d
 
 ```js
 // Normal usage with axios
-let data = (await $axios.get('...')).data
+let data = (await $axios.$get('...')).data
```

Comment:
This lone is valid. normally axios resolves to a response object with data prop ($ version only resolves data